common/src/commands.rs

```diff
@@ -254,13 +306,25 @@ impl TryFrom<proto::CompleteState> for CompleteState {
 
     fn try_from(item: proto::CompleteState) -> Result<Self, Self::Error> {
         Ok(CompleteState {
+            format_version: item
+                .format_version
+                .unwrap_or_else(|| proto::ApiVersion {
+                    version: "unknown_api_version".to_string(),
```

Comment:
I am not sure if this is better option. If I use just `unwrap()` than the test `ankaios_server::tests::utest_server_rejects_update_state_without_format_version` fails. The test simulates the case when the ankaios gets old configuration without format version. This could happen and in this case the server shall not crash. The server shall reject the request as an error ("unknown version") and keep on working.

Reply:
I don't like using a special value. It feels strange to fill in a value which was not there in the original message. I would prefer to just use the empty string in this case. This will also never match an expected version. We can do the same with the JSON decoding.

But if you say you don't like the empty string, I am also OK with the current implementation.

Reply:
This seems to be misunderstanding. I thought that you have suggested to fail (i.e. to use plain `unwrap` instead of `unwrap_or_else`). I am fine with using `unwrap_or_else` and return the empty string (in case the format version is missing). I would suggest to clarify this face to face.

Reply:
Clarified f2f. We have agreed that the best version is to use `unwrap_or_else` and return the empty string.

* The server will gracefully catches error and reject wrong requests.
* The user will get more understandable error message.

Reply:
Fixed